app/validation/date_range_check.py

```diff
@@ -0,0 +1,48 @@
+from app.validation.abstract_validator import AbstractValidator
+from app.validation.validation_result import ValidationResult
+from datetime import datetime
+
+DATE_FROM = "From"
```

Comment:
This will mean that the response labels have to be to and from for a data range question. Is that correct?

What happens when we do a welsh survey?